Source_Code/TM_Backend/TeamMentor.Rest/Controllers/TM_REST_Users.cs

```diff
@@ -257,8 +262,22 @@ public bool				RBAC_IsAdmin()
             }
             return(String.IsNullOrEmpty(errorMessage) ? "Success" : errorMessage);
         }
+        private static bool IsValidEmailAddress(string emailAddress)

```

Comment:
can you make  this an extension method called **valid_Email_Address**(this string emailAddress)